AUTHORS.md

```diff
@@ -0,0 +1,45 @@
+# CB-Geo Material Point Method contributors
+
+> ⚠ This document reflects the contributions of team members as of February 2020.
+
+## MPM Framework
+Krishna Kumar, UT Austin, USA. (krishnak@utexas.edu)
+Shyamini Kularathna, UC Berkeley, USA.
+Ezra Tjung, UC Berkeley, USA. (ezrayst@gmail.com)
+Christopher Wilkes, University of Cambridge, UK.
+Tianchi Zhao, UC Berkeley, USA.
+Thiago Araujo, UT Austin, USA. (thiago.araujo@utexas.edu)
+Bodhinanda Chandra, UC Berkeley, USA.
+Xinyi Qian, UC Berkeley, USA.
+
+## Generalized Interpolation Material Point Method
+Christopher Wilkes, University of Cambridge, UK.
+
+## Domain decomposition / Parallelization schemes
+Krishna Kumar, UT Austin, USA.
+Jeffrey Salmond, University of Cambridge, UK.
+Wentao Xu,  UC Berkeley, USA.
+
+## Two-phase MPM formulation
+Shyamini Kularathna, UC Berkeley, USA.
+Tianchi Zhao, UC Berkeley, USA.
+Weijian Liang, UC Berkeley, USA.
```

Comment:
Is it also good for Weijian to add his HKUST affiliation here?

Reply:
Good idea. Could you add my affiliation and email here? eg.  HKUST, wliangab@ust.hk, Thanks Krishna